crates/rspack_core/src/compiler/hmr.rs

```diff
@@ -92,8 +92,6 @@ where
         self
           .compilation
           .swap_make_artifact_with_compilation(&mut new_compilation);
-        new_compilation.entries = std::mem::take(&mut self.compilation.entries);
```

Comment:
why entries need to be removed

Reply:
Because EntryPlugin will insert entries in hmr, causing 2 duplicated entries